src/FirebaseArduino.h

```diff
@@ -0,0 +1,49 @@
+//
+// Copyright 2016 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+#ifndef FIREBASE_ARDUINO_H
+#define FIREBASE_ARDUINO_H
+
+#include "Firebase.h"
+#include "FirebaseObject.h"
+
+#ifndef FIREBASE_JSONBUFFER_SIZE
+#define FIREBASE_JSONBUFFER_SIZE 200
+#endif // FIREBASE_JSONBUFFER_SIZE
+
+class FirebaseArduino {
+ public:
+  void begin(const char* host, const char* auth = "");
```

Comment:
Since we are presenting this as the super easy way to get into firebase I think we should have a lot of comments on all of the methods here.